### PR TITLE
Invalid void return type

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

`void` can only be used as standalone type.

### Testing Instructions

Review.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: libraries/src/Application/ApplicationHelper.php [hunk 1]
```diff
@@ -171,6 +171,8 @@ public static function getClientInfo($id = null, $byName = false)
                 }
             }
         }
+
+        return null;
```
Comment (ReLater) [inline libraries/src/Application/ApplicationHelper.php:175]:
```suggestion
        return;
```

Comment (SharkyKZ) [reply libraries/src/Application/ApplicationHelper.php:175]: The point here is not to return void because void can't be used in union types.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on a2a5a60d5daea848726da28e77c97102525de622<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39395">issues.joomla.org/tracker/joomla-cms/39395</a>.</sub>

**Comment by Hackwar** [comment]
@ceford "mixed" already contains null. :-)

**Review by Hackwar — APPROVED** [review]
Codereview

**Comment by roland-d** [comment]
Thank you

**Pull request merged**